docs/Fleets.md

```diff
@@ -0,0 +1,68 @@
+# Synctera::Fleets
```

Comment:
These fields now look modeled so this is a potentially breaking change since we're referencing the hash directly.